### PR TITLE
update kubeadm/kinder tests for 1.16

### PR DESCRIPTION
**Pull request by RA489**
update kubeadm/kinder tests for 1.16
fixes https://github.com/kubernetes/kubeadm/issues/1743

### CODE REVIEW

**Comment by neolit123** [comment]
@RA489 me and @fabriziopandini decided that we should not update these versions in the tables every cycle.
they are only examples (under `.e.g` columns).

File: kinder/ci/kubeadm-periodic.tests.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ Upgrade tests are meant to verify the proper functioning of the `kubeadm upgrade
 
 | from                                    | e.g.              | to                                     | e.g.             |
 | --------------------------------------- | ----------------- | -------------------------------------- | ---------------- |
-| current<br />(ci/latest-1.15)           | v1.15.1-alpha...  | master<br />(ci/latest)                | v1.16.0-alpha... |
-| current -1/minor<br />(ci/latest-1.14)  | V1.14.6-alpha...  | current<br />(ci/latest-1.15)          | v1.15.1-alpha... |
-| current -2/minor<br />(ci/latest-1.13)  | V1.13.9-alpha...  | current -2/minor<br />(ci/latest-1.14) | V1.14.5-alpha... |
-| current -3/minor<br />(ci/latest-1.12)  | V1.12.10-alpha... | current -3/minor<br />(ci/latest-1.13) | V1.13.8-alpha... |
+| current<br />(ci/latest-1.16)           | v1.16.1-alpha...  | master<br />(ci/latest)                | v1.17.0-alpha... |
+| current -1/minor<br />(ci/latest-1.15)  | V1.15.6-alpha...  | current<br />(ci/latest-1.16)          | v1.16.1-alpha... |
+| current -2/minor<br />(ci/latest-1.14)  | V1.14.9-alpha...  | current -3/minor<br />(ci/latest-1.15) | V1.15.5-alpha... |
```
Comment (fabriziopandini) [inline kinder/ci/kubeadm-periodic.tests.md]: to current -1 (instead of -3)

File: kinder/ci/kubeadm-periodic.tests.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ Upgrade tests are meant to verify the proper functioning of the `kubeadm upgrade
...
+| current -3/minor<br />(ci/latest-1.13)  | V1.13.10-alpha... | current -4/minor<br />(ci/latest-1.14) | V1.14.8-alpha... |
```
Comment (fabriziopandini) [inline kinder/ci/kubeadm-periodic.tests.md]: to current -2 (instead of -4)

File: kinder/ci/kubeadm-periodic.tests.md (hunk 1 above)
Comment (RA489) [reply kinder/ci/kubeadm-periodic.tests.md]: done

File: kinder/ci/kubeadm-periodic.tests.md (hunk 2 above)
Comment (RA489) [reply kinder/ci/kubeadm-periodic.tests.md]: done

**Comment by RA489** [comment]
@fabriziopandini updated version for discovery tests and kustomize tests. PTAL.

**Pull request merged**